src/components/Profile/MyPosts/MyPosts.jsx

```diff
@@ -5,17 +5,30 @@ import { I18N } from "../../core/constants";
 
 import s from "./MyPosts.module.css";
 
-const MyPosts = () => {
+const MyPosts = (props) => {
+  let postsData = [
+    { id: 1, message: "anti-mage the best", likesCount: 9343 },
+    { id: 2, message: "Salem alem", likesCount: 9333 },
+    { id: 3, message: "asdasd" },
+    { id: 4, message: "asdasd" },
+    { id: 5, message: "asdasdasd" }
+  ];
+
+  let postsElements = postsData.map ( p => 
```

Comment:
Same as for "messagesElements" case.

Reply:
done. 

p/s: such names were given for a more convenient study of the map method